Makefile

```diff
@@ -138,7 +140,7 @@ yamllint:  ## Lint YAML files (does not validate syntax!).
 	@echo
 
 .PHONY: lint
-lint: ansible-config-lint check-ruff app-lint check-black html-lint shellcheck typelint yamllint ## Runs all lint checks
+lint: ansible-config-lint app-lint check-black check-desktop-files check-strings check-ruff check-supported-locales html-lint shellcheck typelint yamllint ## Runs all lint checks
```

Comment:
Sorry I missed this, yes, there was slightly an order! `check-ruff` was second because it was the fastest thing and provided much quicker feedback than all the other tools.

Reply:
Fix proposed in #6995!